variables.tf

```
@@ -66,6 +72,12 @@ variable "truefoundry_db_ingress_cidr_blocks" {
 variable "truefoundry_db_subnet_ids" {
   type        = list(string)
   description = "List of subnets where the RDS database will be deployed"
+  default     = []
+
+  validation {
+    condition     = !var.truefoundry_db_enabled || length(var.truefoundry_db_subnet_ids) > 0
```

Condition should be more understandable
```suggestion
    condition     = var.truefoundry_db_enabled ? length(var.truefoundry_db_subnet_ids) > 0 : true
```